### ISSUE TITLE
Support Add to Home Screen feature for PWA

### ISSUE THREAD

**Issue by iccub** [brave/brave-ios#6962]

**Comment by davidcollini** [comment]
Would this work the same way the Add to Home Screen button on Safari for iOS works?

**Comment by iccub** [comment]
Yes. Although we are are not sure if the implementation is good enough would we get our adblocking and scripts or just a plan webkit web view

**Comment by davidcollini** [comment]
Any updates on this?
